test/testdata/lsp/code_actions/loop_type_change.A.rbedited

```diff
@@ -0,0 +1,8 @@
+# typed: true
+# exhaustive-apply-code-action: true
+
+x = T.let(nil, T.nilable(Integer))
+
+1.times { x = 42 }
+            # ^^ error: Changing the type of a variable in a loop is not permitted
```

Comment:
You can remove these assertion comments.

Reply:
AFAIK, I think that would fail the tests that check equality of expected edited vs. actual edited files.

Reply:
Oof, right